app/src/main/res/values/styles.xml

```diff
@@ -45,4 +45,78 @@
         <item name="android:paddingTop">@dimen/padding_title_bar</item>
         <item name="android:paddingEnd">@dimen/padding_title_bar</item>
     </style>
+
+    <style name="TextView.FullName" parent="TextAppearance.AppCompat">
+        <item name="android:paddingStart">@dimen/padding_title_bar</item>
+        <item name="android:textColor">@color/eclipse</item>
+        <item name="android:textStyle">bold</item>
```

Comment:
`textStyle`, `textColor` `textSize` i `fontFamily` mozna zdefiniowac od razu jako `textAppearance` i wrzucic jeden z gotowych styli przygotowanych przez Bartka. Jeżeli bedziesz chcial jakies dokladne tipy jak to zrobic, daj znac ;)

Reply:
zrobione